src/Datadog.Trace.ClrProfiler.Native/module_metadata.h

```diff
@@ -89,23 +121,46 @@ class ModuleMetadata
 
     void SetWrapperMemberRef(const WSTRING& keyIn, const mdMemberRef valueIn)
     {
-        wrapper_refs[keyIn] = valueIn;
+        std::lock_guard<std::mutex> guard(wrapper_mutex);
```

Comment:
FYI, since C++17, `lock_guard` should be replaced by `scoped_lock` (https://en.cppreference.com/w/cpp/thread/scoped_lock)

Reply:
TIL, done in f18e9da